.github/ISSUE_TEMPLATE/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+<!---
+    Thank you for contributing to Unity.
+    To help us process this pull request we recommend that you add the following information:
+     - Summary and list of changes in the pull request,
+     - Issue(s) related to the changes made such as GitHub or Jira,
+     - Manual testing scenarios if available
+    Fields marked with (*) are required. Please don't remove the template.
+-->
+### Description (*)
+<!---
+    Please provide a description of the changes proposed in the pull request.
+    Letting us know what has changed and why it needed changing will help us validate this pull request.
+-->
+### Related Pull Requests
+<!-- related pull request placeholder -->
+### Issue Number(s) (*)
+<!---
+    Provide a list of fixed issues from Jira (GOMPS-ticketnumber) or GitHub (#issuenumber).
+    This helps us understand the reasoning behind this change, what it fixes, feature being added, etc.
+-->
+Fixes issue(s):
+### Manual testing scenarios
+<!---
+    Please provide a set of unambiguous steps to test the proposed code change.
+    Giving us manual testing scenarios will help with the processing and validation process.
```

Comment:
I think it'd be great mentioning here that if an error was output to the player log, that they should provide that if possible (at least the error logged or even the log file itself).